RulesPacker/saveold/ApplicationInspector.CLI.runtimeconfig.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Same as previous, I don't think these files are supposed to be here.

Reply:
Agreed.  Corrected unintended checkin.